apps/omg_utils/test/omg_utils/http_rpc/validators/base_test.exs

```diff
@@ -160,53 +177,87 @@ defmodule OMG.Utils.HttpRPC.Validator.BaseTest do
              do: {:ok, %{currency: currency, amount: amount}}
       end
 
-      assert {:ok, %{currency: @bin_value, amount: 100}} =
+      {:ok, _address_value} = @params |> Map.get("valid_address") |> Encoding.from_hex()
```

Comment:
This is prepended with an underscore because it somehow does not pick up that the value is used on line `182` and generates a warning?

Reply:
it's not used because you're assigning to it